src/scripts.js

```diff
@@ -46,26 +79,68 @@ function displayAllRecipes() {
 }
 
 function viewRecipeDetail(event) {
-    //GOAL: incoporate instructions and total cost into recipe details page
+  viewRecipeInstructions(event);
+  viewRecipeTotalCost(event);
+  viewRecipeIngredients(event);
```

Comment:
I love that this is helper functions!

Reply:
Unfortunately, you cannot select more than one with radio buttons, so it is an actual radio button that is labeled "All" to get all recipes back. Kiko did this - brilliant! If we choose to do the extension down the road, then we would be able to do multiple select buttons!